### ISSUE TITLE
Prepare training data 

### ISSUE THREAD

**Issue by micrazy** [amzn/pecos#150]
Hi @jiong-zhang , we're testing pecos on our dataset and we got some problem with preparing training data because our limited labelling capabilities. 
So In which case does pecos take better performance?
1. 50000 samples with single label
2. 10000 samples with average 5 labels 

Morever,   Is it necessary to fully label each sample？

tks.

**Comment by jiong-zhang** [comment]
Hi @micrazy, this will depend on what metric/data you use to evaluate your model and the ground truth label distribution of your problem. 
It's not necessary to fully label each sample but those un-labeled (sample, label) pairs will be treated as negative observations. However, it is recommended to have at least one positive observations associated with each label (i.e. no empty columns in label matrix Y).

**Comment by micrazy** [comment]
Thank U for your apply @jiong-zhang !
One more question: In paper, we can see  pecos achieves 88.41% p@1 at Eulex-4K dataset , of which   the number of labels is 3956 and  the average number of labels per instance is 5.3
